keps/sig-network/2438-dual-stack-apiserver/README.md

```diff
@@ -337,8 +400,8 @@ allow the upgrade to proceed smoothly.)
   special-casing `kubernetes.default` as being obligatorily
   single-stack).
 
-- We must be alpha for at least 2 releases, to ensure we don't need to
-  support upgrades from pre-Alpha to Beta.
+- (Maybe) We must be alpha for at least 2 releases, if we don't want
```

Comment:
I don't understand this - do you mean someone would downgrade from 1.25 to 1.23 ?

Reply:
Yes... mentioned in more detail in the "Upgrade / Downgrade Strategy" section; if you downgrade from Beta to 1.23 you may need to do a small amount of manual cleanup.